test/quantization/eager/test_quantize_eager_qat.py

```diff
@@ -51,6 +52,8 @@
     override_qengines,
 )
 
+from torch.testing._internal.common_utils import skipIfNoXNNPACK
```

Comment:
I thought we are going to hide xnnpack entirely, are we still exposing this in torch.backends?

Reply:
yes, that's the goal. I wish I didn't have to add this but for OSS CI there are some builds which don't have USE_XNNPACK, in those cases this test will fail. 

Until XNNPACK is optional we have to do this.